### PR TITLE
Client filters for mongo and memory repos

### PR DESCRIPTION
**Pull request by ht2**

### CODE REVIEW

**Comment by ryansmith94** [comment]
Nice let's see if we can get the tests in with this too.

File: src/tests/getStatement.test.ts [hunk 1]
```diff
@@ -65,4 +76,17 @@ describe('get statement', () => {
     await storeStatements([TEST_STATEMENT, TEST_VOIDER]);
     await assertVoided();
   });
+
+  it('should return the correct statement when the ids are the same across 2 different stores', async () => {
```
Comment (ryansmith94) [inline src/tests/getStatement.test.ts]: Happy with this, just wondered if it should be in the clients.test.ts file?

Comment (ryansmith94) [inline src/tests/getStatement.test.ts]: Ah actually I think I see my confusion you put the clients.test.ts file for storing in the getStatement folder.

File: src/tests/getStatement/clients.test.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import * as assert from 'assert';
```
Comment (ryansmith94) [inline src/tests/getStatement/clients.test.ts]: So I think this file is an unintended duplicate of src/tests/storeStatements/clients.test.ts

Comment (ht2) [inline src/tests/getStatement/clients.test.ts]: Agreed

**Pull request merged**